src/components/TabBar/Tab.tsx

```diff
@@ -0,0 +1,92 @@
+import * as React from 'react'
+import styled, { css } from 'styled-components'
+
+import { InjectedProps, withTheme } from '../../hocs/withTheme'
+import { isTouchDevice } from '../../libs/ua'
+
+export interface Props {
+  id: string
+  label: string
+  selected?: boolean
+  disabled?: boolean
+  className?: string
+  onClick: (tabId: string) => void
+}
+
+type MergedProps = Props & InjectedProps
+
+const TabComponent: React.FC<MergedProps> = ({
+  id,
+  label,
+  onClick,
+  theme,
+  selected = false,
+  className = '',
+  ...props
```

Comment:
[nits] The remaining props are only disabled so there is no need to use the spread operator.